src/platforms/gitlab/release.go

```diff
@@ -70,7 +77,17 @@ func getGitlabBaseUrl(remoteURL string) (gitlabBaseURL string, err error) {
 
 		parts := strings.Split(remoteURL, "/")
 		containsAt := strings.Contains(remoteURL, "@")
-		if len(parts) > 2 {
+		containsHttps := strings.Contains(remoteURL, "https")
```

Comment:
I think we can simplify all this and handle edge cases better with url.Parse() -- Any reason not to use that here? 

https://pkg.go.dev/net/url#Parse

Reply:
Good suggestion, way cleaner now